ipfx/qc_features.py

```diff
@@ -3,15 +3,15 @@
 
 def measure_blowout(v, idx0):
 
-    return 1e3 * np.mean(v[idx0:]*1e-3)
+    return np.mean(v[idx0:])
 
 
 def measure_electrode_0(curr, hz, t=0.005):
 
     n_time_steps = int(t * hz)
     # electrode 0 is the average current reading with zero voltage input
     # (ie, the equivalent of resting potential in current-clamp mode)
-    return 1e12 * np.mean(curr[0:n_time_steps]*1e-12)
+    return np.mean(curr[0:n_time_steps])
```

Comment:
This change and the one in measure_blowout change the exponent from -x to +x. Is that on purpose? If yes can we have a comment for the functions in which units they return their values.

Reply:
Here the result should remains unchanged. Thus, I removed unnecessary conversion.  Looks like we were converting to pA->A, compute mean and then back again to pA . I am not sure why, perhaps @dyf knows.